src/main/webapp/js/instructorCourseEdit.js

```diff
@@ -131,6 +145,13 @@ function checkTheRoleThatApplies(instrNum) {
 	var instrRole = $("#accessControlInfoForInstr" + instrNum + " div div p span").html();
 	$("input[id='instructorroleforinstructor" + instrNum + "']").filter("[value='" + instrRole + "']").prop("checked", true);
 	if (instrRole === "Custom") {
+		// Save original values of Custom Role
+		instructorCourseEditDefaultPrivilegeValuesForCustomRole[instrNum] = [];
+		for (var i = 0; i < instructorPrivilegeValues.length; i++) {
+			var checkValue = $("#tunePermissionsDivForInstructor" + instrNum + " input[name='" + instructorPrivilegeValues[i] + "']").prop("checked");
+			instructorCourseEditDefaultPrivilegeValuesForCustomRole[instrNum][instructorPrivilegeValues[i]] = checkValue;
```

Comment:
https://github.com/airbnb/javascript/tree/master/es5#objects <-- use {} 

Caveat: There are times where we don't use {}. But general rule of the thumb is if you don't know, use {} because if you knew and had a good reason you would know to use new Object() and / or other methods such as the prototypical way of creating an Object.

Read:

http://stackoverflow.com/questions/4597926/what-is-the-difference-between-new-object-and-object-literal-notation

for a more detailed discussion.

Some of the advantages of {} are:

1. It's faster
2. No scope resolution needed
3. Can't be overwritten
4. obj = new Object(1) does not result in a type of "object" but a type of "number" but when you do obj.a = 2, obj is now a type of "object"

and more...